constitution.md

```diff
@@ -0,0 +1,182 @@
+Behavior listed below does not, and cannot encompass every circumstance that
+can be categorized as hazing. Further, this policy is not intended to prohibit
+customary athletic events, contests, and competitions that are sponsored by the
+institution or the organized and supervised practices associated with such
+events or activity or conduct that furthers the goals of a legitimate
+educational curriculum, extracurricular program, or military training program
+as approved by the University.
+
+Hazing activities include but are not limited to; physical abuse; kidnapping,
+blindfolding, confinement, or binding and/or restricting movement; calisthenics
+or other strenuous physical activity used to harass, punish, or harm an
+individual; excursions; spraying, painting, or pelting with any substance;
+burying in any substance; nudity; servitude; exposure to uncomfortable elements;
+verbal abuse or harassment; wearing of apparel which is conspicuous or indecent;
+forcing consumption of any legal or illegal substance (e.g. food, liquid,
+beverage, alcohol, drug, or other substance); depriving of sufficient sleep;
+burning, branding, or tattooing; interrogation in an intimidating or threatening
+manner; misleading members in an effort to convince them that they will not
+become or remain members unless they complete tasks, follow instructions, or act
+in a certain way; misleading members into believing that they will be hurt
+during induction or initiation; carrying items that serve no constructive
+purpose or that are designed to punish or embarrass the carrier; requiring or
+suggesting obtaining, possessing items or completing tasks in an unlawful manner
+(i.e. scavenger hunts); any act that is designed to or likely to compromise the
+dignity of a member or prospective member, cause embarrassment or shame to a
+member or prospective member, cause a member or prospective member to be the
+object of malicious amusement, ridicule, or emotional strain, or cause
+psychological harm or substantial emotional strain; or any other activity which
+may result in physical injury or endanger the health or life of the individual
+being hazed.
+
+An individual cannot consent to being hazed; a victim’s voluntary or willfulr
+participation in hazing activities will not be considered evidence that a
+violation of this policy did not occur. Section 18.2-56 of the Code of Virginia
+declares hazing illegal, establishes conditions for civil and criminal
+liability, and outlines the duties of the university when a student has been
+found guilty of hazing in civil or criminal court.
+
+It is hereby incorporated as part of this constitution and will serve as a guide
+for action by the university and the Unix Users Group if there is an instance
+of hazing by this organization or any of the members of this organization.  It
+will be the duty of the officers of the Unix Users Group to educate the
+membership of this policy.
+
+## Article VIII - Amendments
```

Comment:
Per the email sent by OSAI earlier today:

> Read the information below carefully! It is extremely important for your club. We have changed the language of the required membership statement in your constitutions. You should change it to make sure it matches before uploading your constitutions during the registration process. 

This implies that our constitution may be amended without following this article. Perhaps we can add wording to the end such as:

> In some situations, changes to the constitution may be mandated by the Office of Student Activities and Involvement in order to retain good standing. These changes should be announced at a regular meeting of the organization; however, they do not need to be voted on by the membership.